corehq/apps/locations/views.py

```diff
@@ -261,6 +271,12 @@ class NewLocationView(BaseLocationView):
     creates_new_location = True
     form_tab = 'basic'
 
+    @use_bootstrap3
+    @use_knockout_js
```

Comment:
please pull / rebase latest master into this branch and remove `use_knockout_js` :)